docs/excel/custom-functions-runtime.md

```diff
@@ -66,17 +66,18 @@ ws.onerror = (error) => {
 
 Within a custom function (or within any other part of an add-in), you can store and access data by using the `OfficeRuntime.AsyncStorage` object. `AsyncStorage` is a persistent, unencrypted, key-value storage system that provides an alternative to [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage), which cannot be used within custom functions. An add-in can store up to 10 MB of data using `AsyncStorage`.
 
+`AsyncStorage` is intended as a shared storage solution, meaning multiple parts of an add-in are able to access the same data. For example, tokens for user authentication may be stored in `AsyncStorage` because it can be accessed by both a custom function and add-in UI elements like the task pane. Similarly, if two add-ins share the same domain, they are also permitted to share information back and forth through `AsyncStorage`. However, because `AsyncStorage` can be a shared location, it is important to realize that it is possible to override key-value pairs.
```

Comment:
We know from experience that "domain" is ambiguous to many developers. We need to spell out where different subdomains count as the "same" domain. E.g. Are "funny.contoso.com" and "sad.contoso.com" the same domain or different domains for purposes of AsyncStorage?

Reply:
Good point. I will check with Madhav. My sense is that those are different RE: AsyncStorage but I will double check just to make sure.

Reply:
Checked - yes, diff subdomains are indeed different instances of AsyncStorage. :)